meteo-notebook.ipynb

```diff
@@ -110,10 +110,27 @@
   },
```

Comment:
<p>Can you try putting a semicolon at the end?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rawan-a-d/Air-Quality-Prediction/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
I removed the line that caused the long output